### ISSUE TITLE
Author test case: 5.3.5/6 Required Owned Elements

### ISSUE THREAD

**Issue by cyns** [w3c/aria#1521]
5.3.5/6 Required Owned Elements
When a widget is missing required owned elements due to script execution or loading, authors MUST mark a containing element with aria-busy equal to true.
